website/docs/observability/open-source-metrics/application-metrics.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Custom Metrics"
+sidebar_position: 50
+---
+
+Custom metrics are user defined metrics that help track your application KPIs. These metrics help analyze data and derive insights to meet your application needs. Let's look at some custom metrics using AWS Distro for OpenTelemetry and visualize the metrics using grafana.
+
+The custom metrics we want to capture and show in the dashboard for the sample application are:
+- number of total orders created
+- most popular watch type order, based on number of each watch type ordered
+- total cost of each order
+
+The "retail-store-sample-app" has already been instrumented to capture the above mentioned metrics. The OrderMetrics class uses Spring Boot [Micrometer](https://spring.io/blog/2018/03/16/micrometer-spring-boot-2-s-new-application-metrics-collector) a metrics collection facade which collects metrics data from the application with a vendor neutral API. 
+
+Access the ui in your web browser ,following instructions detailed in workshop under [Exposing applications](https://www.eksworkshop.com/docs/fundamentals/exposing/). Once you load the ui page, add a couple of watch orders so we can generate the metrics.
```

Comment:
You should not refer to other labs. Remove manually loading the application from this section, just use the load generator.

Reply:
done